subsys/bluetooth/host/l2cap.c

```diff
@@ -2154,6 +2154,16 @@ static void le_credits(struct bt_l2cap *l2cap, uint8_t ident,
 	cid = sys_le16_to_cpu(ev->cid);
 	credits = sys_le16_to_cpu(ev->credits);
 
+	if (!L2CAP_LE_CID_IS_DYN(cid)) {
+		LOG_WRN("Can't add credits to non-dynamic channel (cid 0x%04x)", cid);
```

Comment:
Would it be worth logging the pointer to the `l2cap` channel as well?

Reply:
Do you mean `l2cap->chan` or what we get later in the function?
chan = bt_l2cap_le_lookup_tx_cid(conn, cid);
**le_chan** = BT_L2CAP_LE_CHAN(chan);

Reply:
I was just thinking the `l2cap` pointer :) For debugging purposes it's nice to be able to figure out which channel/connection/object that is causing issues